### ISSUE TITLE
Minor Tweaks

### ISSUE THREAD

**Issue by dekanfrus** [jmmcatee/cracklord#79]
The JTR config file (johndict.conf) references hashcat.

![image](https://cloud.githubusercontent.com/assets/2516295/9745173/d84fbba4-5635-11e5-84e3-8905353d6590.png)

The NMAP scan timing and performance is ordered alphabetically rather than numerically.  I think it makes more sense to have it ordered numerically.

![image](https://cloud.githubusercontent.com/assets/2516295/9745052/2f576f6a-5635-11e5-88a7-a5f788903fce.png)

**Comment by emperorcow** [comment]
I completely agree with you on the order, go likes to reorder things sometimes but I will look into that and try and get it into the next build.  I will leave the JTR item to @jmmcatee as he did that one.

**Comment by jmmcatee** [comment]
Hoping to do some cleanup of John in the near future. We have a new form building library that should make fixing the NMap ordering better.
